SourceEditor/app/src/main/res/layout/fragment_item_preview.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think this layout can be replaced easily with a ConstraintLayout, so we don't need then the child LinearLayout, and it's performs better in terms of rendering.